proofs/proof.mli

```diff
@@ -219,7 +219,7 @@ val no_focused_goal : t -> bool
    used. In which case it is [false]. *)
 val run_tactic
   :  Environ.env
-  -> unit Proofview.tactic -> t -> t * (bool*Proofview_monad.Info.tree)
+  -> 'a Proofview.tactic -> t -> t * (bool*Proofview_monad.Info.tree) * 'a
```

Comment:
Can you revert the other API changes and only leave that one? I doubt that the other functions should return values.

Reply:
Changed